.kitchen.docker.yml

```diff
@@ -24,6 +24,7 @@ platforms:
     provision_command:
       - /usr/bin/apt-get update
       - /usr/bin/apt-get install apt-transport-https net-tools -y
+      - mkdir /etc/cron.hourly
```

Comment:
Fair 'nuff! I suppose at some point we should add something to one of the onion layers of the Supermarket install to declare that the cron package is a dependency.